web/src/test/java/org/cbioportal/web/CacheStatsControllerTest.java

```diff
@@ -72,32 +73,46 @@ public static CacheStatisticsService cacheStatisticsService() throws CacheNotFou
 
     @Test
     public void testGetKeysInCache() throws Exception {
-        mockMvc.perform(MockMvcRequestBuilders.get("/" + REPOSITORY_CACHE_ALIAS + "/keysInCache")
-            .accept(MediaType.APPLICATION_JSON))
-            .andExpect(MockMvcResultMatchers.status().isOk())
-            .andExpect(MockMvcResultMatchers.content().contentTypeCompatibleWith(MediaType.APPLICATION_JSON));
+        for (String validCacheAlias : VALID_CACHE_ALIASES) {
+            mockMvc.perform(MockMvcRequestBuilders.get("/" + validCacheAlias + "/keysInCache")
+                .accept(MediaType.APPLICATION_JSON))
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(MockMvcResultMatchers.content().contentTypeCompatibleWith(MediaType.APPLICATION_JSON));
+        }
     }
 
     @Test
     public void testGetKeysInInvalidCache() throws Exception {
-        mockMvc.perform(MockMvcRequestBuilders.get("/" + INVALID_CACHE_ALIAS + "/keysInCache")
+        mockMvc.perform(MockMvcRequestBuilders.get("/" + invalidCacheAlias() + "/keysInCache")
             .accept(MediaType.APPLICATION_JSON))
             .andExpect(MockMvcResultMatchers.status().isNotFound());
     }
 
     @Test
     public void testGetKeyCountsPerClass() throws Exception {
-        mockMvc.perform(MockMvcRequestBuilders.get("/" + REPOSITORY_CACHE_ALIAS + "/keyCountsPerClass")
-            .accept(MediaType.APPLICATION_JSON))
-            .andExpect(MockMvcResultMatchers.status().isOk())
-            .andExpect(MockMvcResultMatchers.content().contentTypeCompatibleWith(MediaType.APPLICATION_JSON));
+        for (String validCacheAlias : VALID_CACHE_ALIASES) {
+            mockMvc.perform(MockMvcRequestBuilders.get("/" + validCacheAlias + "/keyCountsPerClass")
+                .accept(MediaType.APPLICATION_JSON))
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(MockMvcResultMatchers.content().contentTypeCompatibleWith(MediaType.APPLICATION_JSON));
+        }
     }
 
     @Test
     public void testGetKeyCountsPerClassInvalidCache() throws Exception {
-        mockMvc.perform(MockMvcRequestBuilders.get("/" + INVALID_CACHE_ALIAS + "/keyCountsPerClass")
+        mockMvc.perform(MockMvcRequestBuilders.get("/" + invalidCacheAlias() + "/keyCountsPerClass")
             .accept(MediaType.APPLICATION_JSON))
             .andExpect(MockMvcResultMatchers.status().isNotFound());
     }
 
+    private String invalidCacheAlias() {
```

Comment:
Not sure I understand why we changed the valid caches to a list of cache names and added this method to generate an invalid cache name. Wouldn't it be okay to keep the existing tests?

Reply:
agreed ^

Reply:
reverted